<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftBottom</class>
 <widget class="QWidget" name="LeftBottom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>86</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>182</width>
    <height>86</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>182</width>
    <height>86</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">padding:0px;
margin:0px;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetContent" native="true">
     <property name="styleSheet">
      <string notr="true">#widgetContent{
	border:none;
	background-image: url(:/public/left/left_pressure_container.png);
}</string>
     </property>
     <widget class="QLabel" name="labelValue">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>14</y>
        <width>112</width>
        <height>48</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>48</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:48px;
font-weight:blod;
color:#409ab4;</string>
      </property>
      <property name="text">
       <string>-999</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelName">
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>14</y>
        <width>64</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:#409ab4;</string>
      </property>
      <property name="text">
       <string>动脉压</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelUnit">
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>37</y>
        <width>64</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
color:#409ab4;</string>
      </property>
      <property name="text">
       <string>mmHg</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>182</width>
        <height>86</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSlider" name="pressureSlider">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>60</y>
        <width>150</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*horizontal ：水平QSlider*/
QSlider::groove:horizontal {
border: none;
}

/*1.滑动过的槽设计参数*/
QSlider::sub-page:horizontal {
 /*槽颜色*/
background: #54ce7d;
 /*外环区域倒圆角度*/
border-radius: 2px;
 /*上遮住区域高度*/
margin-top:9px;
 /*下遮住区域高度*/
margin-bottom:9px;
/*width在这里无效，不写即可*/
}

/*2.未滑动过的槽设计参数*/
QSlider::add-page:horizontal {
/*槽颜色*/
background: #54ce7d;
/*外环大小0px就是不显示，默认也是0*/
border: 0px solid #777;
/*外环区域倒圆角度*/
border-radius: 2px;
 /*上遮住区域高度*/
margin-top:9px;
 /*下遮住区域高度*/
margin-bottom:9px;
}
 
/*3.平时滑动的滑块设计参数*/
QSlider::handle:horizontal {
/*滑块颜色*/
background: #54ce7d;
/*滑块的宽度*/
width: 10px;
/*滑块外环为1px，再加颜色*/
border: 1px solid #cccdd3;
 /*滑块外环倒圆角度*/
border-radius: 6px; 
 /*上遮住区域高度*/
margin-top:4px;
 /*下遮住区域高度*/
margin-bottom:4px;
}

/*4.手动拉动时显示的滑块设计参数*/
QSlider::handle:horizontal:hover {
/*滑块颜色*/
background: #54ce7d;
/*滑块的宽度*/
width: 10px;
/*滑块外环为1px，再加颜色*/
border: 1px solid #cccdd3;
 /*滑块外环倒圆角度*/
border-radius: 6px; 
 /*上遮住区域高度*/
margin-top:4px;
 /*下遮住区域高度*/
margin-bottom:4px;
}
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <zorder>labelUnit</zorder>
     <zorder>labelValue</zorder>
     <zorder>labelName</zorder>
     <zorder>pressureSlider</zorder>
     <zorder>pushButton</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
